pallets/pool-system/src/impls.rs

```diff
@@ -177,6 +184,10 @@ impl<T: Config> PoolMutate<T::AccountId, T::PoolId> for Pallet<T> {
 				.map_err(|_| Error::<T>::FailedToRegisterTrancheMetadata)?;
 		}
 
+		for (fee_bucket, pool_fee) in pool_fees.into_iter() {
+			T::PoolFees::add_fee(pool_id, fee_bucket, pool_fee)?;
+		}
+
```

Comment:
Why not `PoolRegistry` directly does this work?

Reply:
Related to https://github.com/centrifuge/centrifuge-chain/pull/1633#discussion_r1423980281

Reply:
You save from passing the `pool_fees` parameter then.

Reply:
IIUC, you are suggesting to set pool fees in `PoolRegistry::Call::register` instead of the `<PoolSystem as ModifyPool>::create`?

To me, that approach seems to conflicting the design of the `ModifyPool` trait (impl'd by pool system) which is used to create a pool in the pool registry, when registering a pool. 

Moreover, it would just move the `PoolFees` trait bound from system to registry as the latter does not require it right now.